frontend/build/openapi/openapi-ts.config.ts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig } from "@hey-api/openapi-ts";
+
+// eslint-disable-next-line import/no-default-export -- this library requires a default export
+export default defineConfig({
+  input: "./resources/openapi/openapi.json",
+  output: {
+    path: "frontend/src/metabase-types/openapi",
+    clean: false,
+    format: "prettier",
+  },
+  parser: {
+    filters: {
+      schemas: {
+        // MetabaseLegacyMbqlSchema has circular references that break TS type checking
+        // eslint-disable-next-line no-literal-metabase-strings -- this is not user facing text
+        exclude: ["/^MetabaseLegacyMbqlSchema/"],
+      },
+    },
+  },
+  types: {
+    enums: "javascript", // This generates runtime enums!
+  },
```

Comment:
@chodorowicz this isn't needed, what actually generates runtime enums is that `enums: true` option in `@hey-api/typescript` plugin. Surprised TypeScript doesn't complain here as this isn't even an option anymore!